js/apps/account-ui/src/environment.ts

```diff
@@ -15,6 +19,8 @@ const defaultEnvironment: Environment = {
   isRunningAsTheme: false,
   realm: "master",
   resourceUrl: "http://localhost:8080",
+  logo: "/logo.svg",
+  logoUrl: "",
```

Comment:
Perhaps the default here should be `/` instead?
```suggestion
  logoUrl: "/",
```